ui/src/main/resources/PhenoTips/Clinical Notes Annotation.xml

```diff
@@ -118,377 +118,459 @@
       <cache>forbid</cache>
     </property>
     <property>
-      <code>document.observe('xwiki:dom:loaded', function() {
+      <code>
+document.observe('xwiki:dom:loaded', function() {
 
 // global variables
 
-  /**
```

Comment:
It's not nice to commit whole-file whitespace changes along with real code changes. Separate the two commits please.